### DISCUSSION TITLE
4K视界 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#131] [模板讨论]
4K视界 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by hecarli555** [comment]
近期网站貌似开启了防外观功能，看大佬有没有方法破解。。。？
![image](https://github.com/wjf0214/qd-templates/assets/76683744/e0729e1c-9854-4f8d-a717-c6b33c0c3f6d)

**Reply by wjf0214** [reply]
不影响使用吧。模板签到去签到页面，也就是请求一次。这种应该是防止一个ip挂一堆账号自动签到的。

**Reply by hecarli555** [reply]
我又来了大佬，，，果然4ksj签到也是有问题的。。。显示是成功，但是实际是没签到（k币没增加）如图所示
![image](https://github.com/wjf0214/qd-templates/assets/76683744/b8fe0f6f-9956-47b9-9ed6-ad808c1fdd64)

**Reply by wjf0214** [reply]
更新试试。

**Reply by hecarli555** [reply]
看来他们技术升级了，模板更新了还是不行…
![3813D0C5-A1C0-4393-8FE7-963C200796BD](https://github.com/wjf0214/qd-templates/assets/76683744/4e90739d-f7a6-4140-b498-49c5f1dc19e8)
![B6557409-1FC4-4F16-83B1-4C972393B041](https://github.com/wjf0214/qd-templates/assets/76683744/8dfb9286-3bc0-401a-9fb8-83b3efb4afd2)

**Comment by hecarli555** [comment]
大佬人呢？新版模板依然签到失败，求攻破

**Comment by hecarli555** [comment]
大佬大佬。。。这个模板是不修了么5555

**Comment by hecarli555** [comment]
跪求大佬修复！感谢

**Reply by wjf0214** [reply]
最近忙，而且刚开始那几天网站一直在变，抽空我看看。

**Reply by hecarli555** [reply]
好的，感谢大佬

**Reply by mtyzaq** [reply]
现在隔几天换个验证方式 滑块，验证码，还有个宝塔防御等待3秒，宝塔防御等待3秒这个不知道怎么过验证

**Reply by wjf0214** [reply]
刚刚抽空看了下，还是前几天加上的验证。这几天没看，还是验证轮着来的？和hao4k的验证一样？

**Comment by xuexijiaoben** [comment]
4K视界又可以签到了，大神可以更新下脚本不，现在的似乎不行

**Reply by hecarli555** [reply]
不是改了用app签到？

**Comment by xuexijiaoben** [comment]
> 不是改了用app签到？
现在可以网页签到了

**Reply by hecarli555** [reply]
<html>
<body>

帖主喜欢分享电影与音乐，热衷于折腾；此生希望能够做一个对影音行业、爱好者有贡献有帮助的社区！各位影音发烧友大家好！自我们的APP上线“1年10个月”以来，经过了长时间的测试与使用，大家完全可以放心下载使用。一、签到说明从2022年4月份起，Hao4K影音电脑端签到就暂停了连续签到、及奖励，同时在APP中开起了连续签到和奖励。将从2023年9月14日彻底关闭电脑端签到签到，只能通过APP签到。（不影响4K世界，下载资源）苹果手机不要升级IOS 17我们这么做实数无奈，为了网站能够长久、能够活下去，我们必须这么做。① 近几年观察，网站每日有上万人签到，但实际每日发帖 \ 回帖的连10个都不到。② 因此我们思考、反省，需要提高社区正向氛围、付出者即有收获、不倡导不劳而获。③ 我们更改运营规则，一切围绕“乐于助人的影音发烧友”为中心，重新设计运营规则。④ 基于“人人均可评分”系统，凡你的发帖/回帖能帮助到他人，就有人给你奖励积分。    基于“点赞”“评分”系统，给他人点赞（APP）、评分（PC），同样是可以获得积分。④ 所以APP签到并不会影响“真正的影音发烧友”，反而方便了“发烧友”及须帮助的影音新人。⑤ 我们没有取消“每日签到”获得积分、就可以兑换商品逻辑，反而在APP中增加了积分数量。⑥ APP方面我们提高了交互、注重信息间的连贯性、整合了二手市场等。如果以现有能力没法帮助他人，但是还想获得积分，我们给予一下建议：给他人点赞、给他人评分、分享好的内容到微信，有人阅读就有H币，都是可以获得积分的；同时在APP中签到积分高于电脑端。每日签到：+ 2H币（细则）每日任务：+22H币 和 5活跃分（细则）新人任务：+10H币 和 30活跃分（细则）早晨打卡：+1H币（细则）请大家互相转告与支持！PC签到后续根据情况再次开放！H币是什么、有什么用？怎么获得https://www.hao4k.cn/plugin.php?id=keke_help&hid=10二、签到入口继续原有签到：（9月14日彻底关闭原签到入口）手机APP签到：https://www.hao4k.cn/plugin.php?id=keke_help&hid=45 | 各位影音发烧友大家好！自我们的APP上线“1年10个月”以来，经过了长时间的测试与使用，大家完全可以放心下载使用。一、签到说明从2022年4月份起，Hao4K影音电脑端签到就暂停了连续签到、及奖励，同时在APP中开起了连续签到和奖励。将从2023年9月14日彻底关闭电脑端签到签到，只能通过APP签到。（不影响4K世界，下载资源）苹果手机不要升级IOS 17我们这么做实数无奈，为了网站能够长久、能够活下去，我们必须这么做。① 近几年观察，网站每日有上万人签到，但实际每日发帖 \ 回帖的连10个都不到。② 因此我们思考、反省，需要提高社区正向氛围、付出者即有收获、不倡导不劳而获。③ 我们更改运营规则，一切围绕“乐于助人的影音发烧友”为中心，重新设计运营规则。④ 基于“人人均可评分”系统，凡你的发帖/回帖能帮助到他人，就有人给你奖励积分。    基于“点赞”“评分”系统，给他人点赞（APP）、评分（PC），同样是可以获得积分。④ 所以APP签到并不会影响“真正的影音发烧友”，反而方便了“发烧友”及须帮助的影音新人。⑤ 我们没有取消“每日签到”获得积分、就可以兑换商品逻辑，反而在APP中增加了积分数量。⑥ APP方面我们提高了交互、注重信息间的连贯性、整合了二手市场等。如果以现有能力没法帮助他人，但是还想获得积分，我们给予一下建议：给他人点赞、给他人评分、分享好的内容到微信，有人阅读就有H币，都是可以获得积分的；同时在APP中签到积分高于电脑端。每日签到：+ 2H币（细则）每日任务：+22H币 和 5活跃分（细则）新人任务：+10H币 和 30活跃分（细则）早晨打卡：+1H币（细则）请大家互相转告与支持！PC签到后续根据情况再次开放！H币是什么、有什么用？怎么获得https://www.hao4k.cn/plugin.php?id=keke_help&hid=10二、签到入口继续原有签到：（9月14日彻底关闭原签到入口）手机APP签到：https://www.hao4k.cn/plugin.php?id=keke_help&hid=45
-- | --
各位影音发烧友大家好！自我们的APP上线“1年10个月”以来，经过了长时间的测试与使用，大家完全可以放心下载使用。一、签到说明从2022年4月份起，Hao4K影音电脑端签到就暂停了连续签到、及奖励，同时在APP中开起了连续签到和奖励。将从2023年9月14日彻底关闭电脑端签到签到，只能通过APP签到。（不影响4K世界，下载资源）苹果手机不要升级IOS 17我们这么做实数无奈，为了网站能够长久、能够活下去，我们必须这么做。① 近几年观察，网站每日有上万人签到，但实际每日发帖 \ 回帖的连10个都不到。② 因此我们思考、反省，需要提高社区正向氛围、付出者即有收获、不倡导不劳而获。③ 我们更改运营规则，一切围绕“乐于助人的影音发烧友”为中心，重新设计运营规则。④ 基于“人人均可评分”系统，凡你的发帖/回帖能帮助到他人，就有人给你奖励积分。    基于“点赞”“评分”系统，给他人点赞（APP）、评分（PC），同样是可以获得积分。④ 所以APP签到并不会影响“真正的影音发烧友”，反而方便了“发烧友”及须帮助的影音新人。⑤ 我们没有取消“每日签到”获得积分、就可以兑换商品逻辑，反而在APP中增加了积分数量。⑥ APP方面我们提高了交互、注重信息间的连贯性、整合了二手市场等。如果以现有能力没法帮助他人，但是还想获得积分，我们给予一下建议：给他人点赞、给他人评分、分享好的内容到微信，有人阅读就有H币，都是可以获得积分的；同时在APP中签到积分高于电脑端。每日签到：+ 2H币（细则）每日任务：+22H币 和 5活跃分（细则）新人任务：+10H币 和 30活跃分（细则）早晨打卡：+1H币（细则）请大家互相转告与支持！PC签到后续根据情况再次开放！H币是什么、有什么用？怎么获得https://www.hao4k.cn/plugin.php?id=keke_help&hid=10二、签到入口继续原有签到：（9月14日彻底关闭原签到入口）手机APP签到：https://www.hao4k.cn/plugin.php?id=keke_help&hid=45


</body>
</html>[帖主](https://www.hao4k.cn/thread-76016-1-1.html)
喜欢分享电影与音乐，热衷于折腾；此生希望能够做一个对影音行业、爱好者有贡献有帮助的社区！
各位影音发烧友大家好！
自我们的APP上线“1年10个月”以来，经过了长时间的测试与使用，大家完全可以放心下载使用。

一、签到说明
从2022年4月份起，Hao4K影音电脑端签到就暂停了连续签到、及奖励，同时在APP中开起了连续签到和奖励。
将从2023年9月14日彻底关闭电脑端签到签到，只能通过[APP签到](https://www.hao4k.cn/plugin.php?id=keke_help&hid=45)。（[不影响4K世界，下载资源](http://www.4ksj.com/)）

苹果手机不要升级IOS 17

我们这么做实数无奈，为了网站能够长久、能够活下去，我们必须这么做。
① 近几年观察，网站每日有上万人签到，但实际每日发帖 \ 回帖的连10个都不到。
② 因此我们思考、反省，需要提高社区正向氛围、付出者即有收获、不倡导不劳而获。
③ 我们更改运营规则，一切围绕“乐于助人的影音发烧友”为中心，重新设计运营规则。
④ 基于“人人均可评分”系统，凡你的发帖/回帖能帮助到他人，就有人给你奖励积分。
    基于“点赞”“评分”系统，给他人点赞（APP）、评分（[PC](https://www.hao4k.cn/home.php?mod=task)），同样是可以获得积分。
④ 所以APP签到并不会影响“真正的影音发烧友”，反而方便了“发烧友”及须帮助的影音新人。
⑤ 我们没有取消“每日签到”获得积分、就可以兑换商品逻辑，反而在APP中增加了积分数量。
⑥ APP方面我们提高了交互、注重信息间的连贯性、整合了二手市场等。

如果以现有能力没法帮助他人，但是还想获得积分，我们给予一下建议：
给他人点赞、给他人评分、分享好的内容到微信，有人阅读就有H币，都是可以获得积分的；同时在APP中签到积分高于电脑端。
每日签到：+ 2H币（[细则](https://www.hao4k.cn/plugin.php?id=keke_help&hid=46)）
每日任务：+22H币 和 5活跃分（[细则](https://www.hao4k.cn/plugin.php?id=keke_help&hid=36)）
新人任务：+10H币 和 30活跃分（[细则](https://www.hao4k.cn/plugin.php?id=keke_help&hid=37)）
早晨打卡：+1H币（[细则](https://www.hao4k.cn/plugin.php?id=keke_help&hid=43)）
请大家互相转告与支持！PC签到后续根据情况再次开放！
H币是什么、有什么用？怎么获得
https://www.hao4k.cn/plugin.php?id=keke_help&hid=10

二、签到入口
继续原有签到：（9月14日彻底关闭原签到入口）
手机APP签到：https://www.hao4k.cn/plugin.php?id=keke_help&hid=45

**Reply by hecarli555** [reply]
????哪里能网页签到了。。。。

**Reply by Misuzu7** [reply]
这里是4ksj，不是hao4k的反馈区

**Reply by hecarli555** [reply]
搞错了，，，不过4ksj也没看到哪里能签到啊
